app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -36,6 +38,10 @@ object GameState {
 
     // Constants
     private const val BASE_URL = "http://158.101.160.143:42069"
+    const val STARTING_FUNDS_DEFAULT = 1000
+    const val SMALL_BLIND_DEFAULT = 100
+    const val MAX_PLAYERS = 8
+    const val MAX_NICKNAME_LEN = 20
```

Comment:
Wouldn't it be better if it was in Player class instead?

Reply:
Well I know what you mean but I don't think it is related to game state that much and it references directly player's property. I know it will be in different files but is it actually bad?
I mean, it seems to be more natural to call `Player.MAX_NICKNAME_LEN` than `GameState.MAX_NICKNAME_LEN`.

Reply:
Agreed, I'll move it into the `Player` class